### ISSUE TITLE
[bug] grpc reflection not working

### ISSUE THREAD

**Issue by ghstahl** [centrifugal/centrifugo#826]
I use postman to make grpc calls and use reflection on all our internal golang services.  

```Feature Request```: ***Log entire config, with secrets redacted at startup.***

```json
{
  "grpc_api": true,
  "grpc_api_address": "localhost",
  "grpc_api_port": 50051,
  "grpc_api_reflection": true,
  "grpc_api_tls_disable": true,
  "token_jwks_public_endpoint": "http://mock-oauth2-pro:50053/.well-known/jwks",
  "admin_password": "password",
  "admin_secret": "secret",
  "admin": true,
  "api_key": "secret",
  "allowed_origins": ["http://localhost:3000"],
  "engine": "redis",
  "redis_address": ["redis:6379"],
  "namespaces": [
    {
      "name": "connector",
      "presence": true,
      "join_leave": true,
      "history_size": 200,
      "history_ttl": "300h",
      "force_positioning": true,
      "force_recovery": true,
      "allow_subscribe_for_client": true,
      "allow_subscribe_for_anonymous": true,
      "allow_publish_for_subscriber": true,
      "allow_publish_for_anonymous": true,
      "allow_history_for_subscriber": true,
      "allow_history_for_anonymous": true,
      "allow_presence_for_subscriber": true,
      "allow_presence_for_anonymous": true
    },
    {
      "name": "connector_private",
      "presence": true,
      "join_leave": true,
      "history_size": 200,
      "history_ttl": "300h",
      "force_positioning": true,
      "force_recovery": true
    }
  ]
}
```
## Startup Logs
```txt
2024-06-06 09:47:18 {"level":"warn","time":"2024-06-06T16:47:18Z","message":"Centrifugo PRO license not found, run in sandbox mode: max connections 20, max nodes 2, max api rps 5"}
2024-06-06 09:47:18 {"level":"info","version":"5.4.0","release_time":"1717095635","edition":"pro","runtime":"go1.22.3","pid":1,"engine":"redis","gomaxprocs":12,"time":"2024-06-06T16:47:18Z","message":"starting Centrifugo"}
2024-06-06 09:47:18 {"level":"info","owner":"unlicensed","max_connections":20,"max_nodes":2,"max_api_rps":5,"time":"2024-06-06T16:47:18Z","message":"license information"}
2024-06-06 09:47:18 {"level":"info","path":"/centrifugo/config.json","time":"2024-06-06T16:47:18Z","message":"using config file"}
2024-06-06 09:47:18 {"level":"info","time":"2024-06-06T16:47:18Z","message":"serving GRPC API service on localhost:50051"}
2024-06-06 09:47:18 {"level":"info","time":"2024-06-06T16:47:18Z","message":"serving websocket, api, admin endpoints on :8000"}
```
<img width="946" alt="image" src="https://github.com/centrifugal/centrifugo/assets/201257/3117a32c-fd17-4fab-8961-4f12b4654cef">

I can make calls to the ```api/info``` endpoint.  
```curl
curl --location 'http://localhost:8765/api/info' \
--header 'accept;' \
--header 'Accept: application/json' \
--header 'X-API-Key: secret' \
--header 'Content-Type: application/json' \
--data '{}'
```

<img width="796" alt="image" src="https://github.com/centrifugal/centrifugo/assets/201257/94eae7c6-3d97-4a9d-86ff-1f63eda67300">

**Versions**
```
"version":"5.4.0"
```

## Docker-compose services
```yaml
  redis:
    image: redis:${REDIS_VERSION:-7}-alpine
    ports:
      - "6379:6379"
    healthcheck:
      test: ["CMD", "redis-cli", "ping"]
  sentinel:
    image: redis:${REDIS_VERSION:-7}-alpine
    entrypoint:
      - /bin/sh
      - -c
      - |
        redis-server --save "" --appendonly no --port 6380 &
        echo "sentinel monitor mymaster 127.0.0.1 6380 2\n" > sentinel.conf
        redis-server sentinel.conf --sentinel
    ports:
      - "6380:6380"
      - "26379:26379"
  cluster:
    image: redis:${REDIS_VERSION:-7}-alpine
    entrypoint:
      - /bin/sh
      - -c
      - |
        redis-server --port 7000 --save "" --appendonly no --cluster-enabled yes --cluster-config-file 7000.conf &
        redis-server --port 7001 --save "" --appendonly no --cluster-enabled yes --cluster-config-file 7001.conf &
        redis-server --port 7002 --save "" --appendonly no --cluster-enabled yes --cluster-config-file 7002.conf &
        while ! redis-cli --cluster create 127.0.0.1:7000 127.0.0.1:7001 127.0.0.1:7002 --cluster-yes; do sleep 1; done
        wait
    ports:
      - "7000:7000"
      - "7001:7001"
      - "7002:7002"

  centrifugo-pro:
    container_name: centrifugo-pro
    image: centrifugo/centrifugo-pro:v5
    volumes:
      - ./configs/centrifugo/config-pro.json:/centrifugo/config.json
    command: centrifugo -c config.json
    ports:
      - 8765:8000
      - 50051:50051
    ulimits:
      nofile:
        soft: 65535
        hard: 65535
    depends_on:
      redis:
        condition: service_healthy
```

**Comment by FZambia** [comment]
Hello @ghstahl 

I just tried – and it works for me with Postman and grpcurl

<img width="938" alt="Screenshot 2024-06-07 at 07 54 24" src="https://github.com/centrifugal/centrifugo/assets/1196565/81b68ea0-cac2-48f6-abf3-eb60cbf5b442">

```
❯ grpcurl -plaintext localhost:50051 list centrifugal.centrifugo.api.CentrifugoApi
centrifugal.centrifugo.api.CentrifugoApi.Batch
centrifugal.centrifugo.api.CentrifugoApi.BlockUser
centrifugal.centrifugo.api.CentrifugoApi.Broadcast
centrifugal.centrifugo.api.CentrifugoApi.CancelPush
...
```

Probably you need to update Postman version?

**Comment by ghstahl** [comment]
I see this problem on 2 different machines.
I can do a reflection call to my micro but not centrifugo.  

can you use my docker-compose to see if you can repo this on your end.  
My [test project](https://github.com/fluffy-bunny/centrifugo-play).  

```bash
docker-compose -f ./docker-compose-pro.yml pull
docker-compose -f ./docker-compose-pro.yml up
```

## postman
```
Postman for Windows
Version
11.1.25
UI version
11.1.25-ui-240604-0453

Desktop platform version
11.1.14

Architecture
x64

OS platform
win32 10.0.22631
```
## docker
```
4.31.0
```
Latest docker on latest windows 11 pro.
I am seeing this error as well out of postgres.
```
Could not load server reflection.
Details: 13 INTERNAL: grpc: failed to unmarshal the received message: failed to unmarshal, message is *grpc_reflection_v1alpha.ServerReflectionRequest (missing vtprotobuf helpers)
```

This works on my grpc micro service.
![image](https://github.com/centrifugal/centrifugo/assets/201257/5da79520-ca42-444d-a1f1-356c9383d71a)

## Centrifugo Pro Logs
```
2024-06-07 14:39:55 {"level":"warn","time":"2024-06-07T21:39:55Z","message":"Centrifugo PRO license not found, run in sandbox mode: max connections 20, max nodes 2, max api rps 5"}
2024-06-07 14:39:55 {"level":"info","version":"5.4.0","release_time":"1717095635","edition":"pro","runtime":"go1.22.3","pid":1,"engine":"redis","gomaxprocs":20,"time":"2024-06-07T21:39:55Z","message":"starting Centrifugo"}
2024-06-07 14:39:55 {"level":"info","owner":"unlicensed","max_connections":20,"max_nodes":2,"max_api_rps":5,"time":"2024-06-07T21:39:55Z","message":"license information"}
2024-06-07 14:39:55 {"level":"info","path":"/centrifugo/config.json","time":"2024-06-07T21:39:55Z","message":"using config file"}
2024-06-07 14:39:55 {"level":"info","time":"2024-06-07T21:39:55Z","message":"serving GRPC API service on localhost:50051"}
2024-06-07 14:39:55 {"level":"info","time":"2024-06-07T21:39:55Z","message":"serving websocket, api, admin endpoints on :8000"}
```

**Comment by FZambia** [comment]
> Centrifugo Pro Logs

Ah, I see it's a PRO version, I missed it originally. Reproduced the problem - will be fixed in next release.

**Comment by FZambia** [comment]
Should be fixed in [Centrifugo PRO v5.4.1](https://github.com/centrifugal/centrifugo-pro/releases/tag/v5.4.1). Many thanks for the report!
